src/main.c

```diff
@@ -32,7 +32,7 @@
 #include "render_courses.h"
 #include "actors.h"
 #include "staff_ghosts.h"
-#include <defines.h>
```

Comment:
Deleted the defines.h header from main.c. Is this intentional?

Reply:
Yes, it was defined twice